src/components/Calculator.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
```

Comment:
- [ ] Your calculator does not work after refactoring. Pls fix this:

https://www.loom.com/share/dfc42a94c1fd4f6c8973cab78e3468ef

Reply:
@topeogunleye Good Morning. Can you please help out with this. I've tried everything possible. Can't find what is wrong. If you can help me point out the fault. That will be much appreciated. Thank you

Reply:
> @topeogunleye Good Morning. Can you please help out with this. I've tried everything possible. Can't find what is wrong. If you can help me point out the fault. That will be much appreciated. Thank you

Hi @StarMindz, sure no problem. Please send me a direct message on slack.